tests/functional/aws-node-sdk/package.json

```diff
@@ -20,7 +21,7 @@
   },
   "scripts": {
     "test-aws": "AWS_ON_AIR=true mocha",
-    "test": "mocha",
+    "test": "mocha -t 50000",
```

Comment:
Because when I writing the data on S3 during the before, it's take more than 10000 ms because I need to add more than 1000 entries.